### ISSUE TITLE
Deleted items are duplicated after pressing Refresh if Save was not pressed before it

### ISSUE THREAD

**Issue by kseniiaguzeeva** [dbeaver/dbeaver#10544]
#### System information: 
- Operating system (distribution) and version Win10
- DBeaver version 7.3.0 EA

#### Steps to reproduce, if exist:

1. Open a table, click a column, then press Delete and Ok in the confirmation popup
2. Press Refresh

Expected: the deletion is reverted
Actual result: the deleted objects appear and their number is doubled
![YNhD0Hg5Py](https://user-images.githubusercontent.com/51405061/100544363-2bedf480-3266-11eb-8970-4ec95fe9d707.gif)

**Comment by flambchap** [comment]
Fixed.

**Comment by flambchap** [comment]
I want to point out here (just for the history) one small thing. The fix itself is trivial. At some point, the line `commandContext.resetChanges(false)` had been changed to `commandContext.resetChanges(true)`, despite the fact that the comment above that line explains why the argument should be `false`. So, I've reverted that change. But neither I nor Serge was able to tell from the history of commits why the change happened.
